tools/manifest/manifest.py

```diff
@@ -20,6 +20,13 @@ class ManifestVersionMismatch(ManifestError):
     pass
 
 
+def iterfilter(filters, iter):
```

Comment:
Maybe call this `tests_iter` or something.

Reply:
I stole this from wptrunner; didn't seem worth it to try to find a place to share the code, but might as well keep the name

Reply:
OK